packages/chart/src/types/ChartConfig.ts

```diff
@@ -68,6 +68,7 @@ type DataFormat = {
   roundTo: number
   suffix: string
   onlyShowTopPrefixSuffix?: boolean
+  roundingPercentage: string
```

Comment:
what's the difference between roundingPercentage and roundTo? can roundTo be reused?